code/src/index.css

```diff
@@ -1,13 +1,160 @@
+*{
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: Arial, Helvetica, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+}
+
+a {
+  text-decoration:none;
+}
+
+/* Mobile first  */
+.album-parent {
+  display: flex;
+  flex-wrap: wrap;
+  flex-grow: 1;
+  padding: 5%;
+  justify-content: space-between;
+}
+
+.album-child {
+  width: 100%;
+  margin-bottom: 30px;
+}
+
+.album-child img{
+  width: 100%;
+}
+
+.cover-container {
+  position: relative;
+}
+
+.artist-name {
+  display: block;
+}
+
+.icon-play {
+  height: 60px;
+  width: 60px;
+}
+
+.icon-dots, .icon-heart {
+  height: 30px;
+  width: 30px;
+  /* display: none; */
+} 
+
+.icon-play:hover {
+  height: 80px;
+  width: 80px;
+  transition: .3s ease;
+  fill: white;
+}
+
+.overlay {
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  bottom:0;
+  left: 0;
+  right:0;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  background: rgba(0, 0, 0, 0.5);
+}
+
+.overlay:hover {
+  opacity: 1;
+}
+
+a {
+  color: white;
+}
+
+h1 {
+  color: #ffffff;
+  padding-left: 5%;
+  padding-top: 3%;
+}
```

Comment:
Tips!
To add underline headline add 
border-bottom: 2px solid rgba(255, 255, 255, 0.1);
to h1.